### PR TITLE
Develop 6.7 merge up to 7.0

### PR DESCRIPTION
**Pull request by livecodefraser**
Merge was clean except for submodules (which were themselves clean).

### CODE REVIEW

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 2a90c5a

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 2a90c5a237c392e61289a8e80fd335739216bf4e

**Comment by livecode-vulcan** [comment]
😞 test failure 2a90c5a237c392e61289a8e80fd335739216bf4e
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/503)
- try-win-x86: [pending](https://vulcan.livecode.com/builders/try-win-x86/builds/561)
- try-ios: [failure](https://vulcan.livecode.com/builders/try-ios/builds/449)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/514)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/449)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/440)

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 1372692

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 137269252e673b835f87f655d2d1976eebb9f33b

File: revspeech/src/revspeech.cpp [hunk 1]
```diff
@@ -75,9 +76,14 @@ static bool NarratorUnload(void)
 
 ///////////////////////////////////////////////////////////////////////////////
 
+extern "C" void getXtable();
+
 void revSpeechLoad(char *args[], int nargs, char **retstring,
 					Bool *pass, Bool *error)
 {
+	// Force a reference to the getXtable function
+	getXtable();
```
Comment (runrevmark) [inline revspeech/src/revspeech.cpp]: This might be a bad idea - getXtable() takes parameters which are pointers to return values in...

**Comment by livecode-vulcan** [comment]
😎 test success 137269252e673b835f87f655d2d1976eebb9f33b
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/504)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/562)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/450)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/515)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/450)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/441)

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay db31a19

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok db31a19c6010cf2eedef4edee3ea8b33adca4d3a

**Comment by livecode-vulcan** [comment]
😎 test success db31a19c6010cf2eedef4edee3ea8b33adca4d3a
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/507)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/565)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/453)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/518)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/453)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/444)

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 05e60e1

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 05e60e1a51c3ef845158fdacd03f82a9d53a516e

**Comment by livecode-vulcan** [comment]
😞 test failure 05e60e1a51c3ef845158fdacd03f82a9d53a516e
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/512)
- try-win-x86: [failure](https://vulcan.livecode.com/builders/try-win-x86/builds/570)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/458)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/523)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/458)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/449)

**Comment by livecodesebastien** [comment]
@livecode-vulcan retry 05e60e1

**Comment by livecode-vulcan** [comment]
😎 test success 05e60e1a51c3ef845158fdacd03f82a9d53a516e
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/513)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/571)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/459)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/524)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/459)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/450)

**Pull request merged**